src/tokenizer/tokenizer.py

```diff
@@ -38,32 +38,16 @@
 
 """
 
-from typing import (
-    Any,
-    List,
-    Mapping,
-    FrozenSet,
-    Callable,
-    Optional,
-    Iterator,
-    Iterable,
-    Tuple,
-    Deque,
-    Union,
-    Match,
-    TypeVar,
-    Type,
-    cast,
-)
-
-import re
 import datetime
+import re
 import unicodedata  # type: ignore
 from collections import deque
+from typing import (Any, Callable, Deque, FrozenSet, Iterable, Iterator, List,
```

Comment:
By convention, typing imports are put at the very top of each source file.

Reply:
Is this a convention in `tokenizer` or in general?

Reply:
At least in Tokenizer, Greynir, GreynirPackage, Ratatoskur.

Reply:
For the record, isort does not follow this convention and orders typing imports like any other import.